monitoring/src/check-page/common-functions.ts

```diff
@@ -46,33 +47,94 @@ export const makeNewBrowser = async (debugMode: boolean): Promise<Browser> => {
 	return browser;
 };
 
+export const openPrivacySettingsPanel = async (config: Config, page: Page) => {
+	log_info(`Loading privacy settings panel: Start`);
+	// Ensure that Sourcepoint has enough time to load the CMP
+	await page.waitForTimeout(3000);
+
+	const frame = getFrame(page, config.iframeDomain);
+	await frame.click(ELEMENT_ID.TCFV2_FIRST_LAYER_MANAGE_COOKIES);
+
+	log_info(`Loading privacy settings panel: Complete`);
+};
+
+export const checkPrivacySettingsPanelIsOpen = async (
+	config: Config,
+	page: Page,
+): Promise<void> => {
+	await page.waitForTimeout(3000);
+	log_info(`Waiting for Privacy Settings Panel: Start`);
+	const frame = getFrame(page, config.iframeDomainSecondLayer);
+	await frame.waitForSelector(ELEMENT_ID.TCFV2_SECOND_LAYER_HEADLINE);
+	log_info(`Waiting for Privacy Settings Panel: Complete`);
+};
+
+export const clickSaveAndCloseSecondLayer = async (
+	config: Config,
+	page: Page,
+) => {
+	log_info(`Clicking on save and exit button: Start`);
+	// Ensure that Sourcepoint has enough time to load the CMP
```

Comment:
Why is this value different from the above timeout?

Reply:
I think you're right, we were testing out a few things and I suspect those were accidentally left.  All now set the same unless there's a particular reason.